functions/Get-DbaSqlService.ps1

```diff
@@ -75,84 +80,39 @@ PROCESS
             if ( $Server.ComputerName )
             {
                 $Computer = $server.ComputerName
-                Write-Verbose "$FunctionName - Getting SQL Server namespace on $Computer via CIM (WSMan)"
-                $namespace = Get-CimInstance -ComputerName $Computer -NameSpace root\Microsoft\SQLServer -ClassName "__NAMESPACE" -Filter "Name Like 'ComputerManagement%'" -ErrorAction SilentlyContinue |
-                            Where-Object {(Get-CimInstance -ComputerName $Computer -Namespace $("root\Microsoft\SQLServer\" + $_.Name) -Query "SELECT * FROM SqlService" -ErrorAction SilentlyContinue).count -gt 0} |
+                Write-Message -Level Verbose -Message "Getting SQL Server namespace on $Computer"
+                $namespace = Get-DbaCmObject -ComputerName $Computer -NameSpace root\Microsoft\SQLServer -Query "Select * FROM __NAMESPACE WHERE Name Like 'ComputerManagement%'" -ErrorAction SilentlyContinue |
```

Comment:
Don't know if we should add a `Resolve-DbaNetworkName` as we do that in some of the other commands that call DbaCmObject?

As well, line 71 `$FunctionName` can be removed (appears to still be there from GitHub view).